R/tbl_continuous.R

```diff
@@ -61,9 +69,23 @@ tbl_continuous <- function(data,
   include <- setdiff(include, c(by, variable)) # remove by and variable columns from list vars included
   data <- dplyr::ungroup(data) |> .drop_missing_by_obs(by = by) # styler: off
 
+  # assign types and values
+  cards::process_formula_selectors(data[include], value = value)
+  type <- rep_named(include, list("categorical")) |>
+    utils::modifyList(rep_named(names(compact(value)), list("dichotomous")))
```

Comment:
This adds the dichotomous values to type list, right?

Reply:
yep!